src/models/generated/ReserveDemandCurve.jl

```diff
@@ -18,7 +17,6 @@ Data Structure for a operating reserve with demand curve product for system simu
 - `name::String`
 - `available::Bool`
 - `time_frame::Float64`: the relative saturation time_frame, validation range: `(0, nothing)`, action if invalid: `error`
-- `operation_cost::TwoPartCost`: Cost for providing reserves  [`TwoPartCost`](@ref)
```

Comment:
should this have a variable cost like the market bid and have the same functions add/set/ etc ?

Reply:
Yup it should